docs/_sass/color_schemes/awsstyle.scss

```diff
@@ -0,0 +1,185 @@
+@import "./color_schemes/dark";
+
+//############ Body ############
+
+$white: #E7E7E7;
+$grey-dk-300: #0C0C0D;
+$link-color: #71A621;
+$body-background-color: #0C0C0D;
+$sidebar-color:#0C0C0D;
+$body-font-family: "Amazon Ember", system-ui, -apple-system, blinkmacsystemfont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Segoe UI Emoji" !default;
+$border-color: #272727;
+
+//Regular
+@font-face {
+	font-family: 'Amazon Ember';
+	src:url('/fonts/amazonember_w_rg.woff2') format('woff2');
+	font-weight:normal;
+	font-style:normal;
+}
+@font-face {
+	font-family: 'Vault Alarm';
+        src:url('/fonts/vaultalarm-regular.otf') format('opentype');
+        font-weight:normal;
+        font-style:normal;
+}
+
+
+//############ _variables.scss ############
+//$body-font-family: "Amazon Ember", system-ui, -apple-system, blinkmacsystemfont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Segoe UI Emoji" !default;
+$body-font-family: "Amazon Ember", system-ui, -apple-system, blinkmacsystemfont, sans-serif, !default;
+
+$content-width: 30rem !default;
+$header-height: 5rem !default;
+
+
+
+//############ Classes ############
+
+ body {
+	//overflow-y:none !important;
+	 position:fixed;
+	 font-family: "Amazon Ember";
```

Comment:
we need another !important hack here so the body is actually in ember, or we can move this to a separate selector with higher specificity like `body *`

Reply:
Done, thanks!